generics/helpers/kafkaCommunications.js

```
@@ -298,6 +304,32 @@ const pushUserCoursesToKafka= function (message) {
+			let kafkaPushStatus = await pushMessageToKafka([
+				{
+					topic: pushDeletedResourceTopic,
+					messages: JSON.stringify(message),
+				},
+			])
+
+			return resolve(kafkaPushStatus)
+		} catch (error) {
+			return reject(error)
+		}
+	})
```

Biome (2.1.2)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)